src/utilities/getIndicatorTableData.js

```diff
@@ -75,16 +92,26 @@ const getIndicatorTableColumns = (indicator, period) => {
   const descriptionColumn = COLUMNS.DESCRIPTION;
   const codeColumn = COLUMNS.CODE;
   // eslint-disable-next-line no-unused-vars
-  const valueColumns = indicator.columns.map(({ description, code, valueType }) => ({
+  const valueColumns = indicator.columns.map(({ description, code }) => ({
+    ...COLUMN_INDICATOR_MUTABLE,
     title: description,
     key: description,
-    type: valueType,
-    ...COLUMN_DEFAULTS,
   }));
 
   return [descriptionColumn, codeColumn, ...valueColumns];
```

Comment:
props code column first

Reply:
The IC blueprint we are working off has the columns in this order. Having said that, columns could benefit from having have a separate unique code which is not identical to their regimen code, but I think that is a smaller issue which can be resolved later.